package.json

```diff
@@ -46,9 +46,13 @@
     "test-rooms-disabled": "ENABLE_ROOMS=false HIDDEN_ROUTES=/rooms mocha -R dot --recursive tests/roomFlag/roomFlagDisabled",
     "test-db": "npm run checkdb && mocha -R dot --recursive tests/db",
     "test-cors": "CORS_ROUTES=/v1/bots,/v1/botActions,/v1/botData,/v1/events,/v1/rooms mocha -R dot --recursive tests/cors",
-    "test-cache": "mocha -R dot --recursive tests/cache",
+    "test-jobQueue": "mocha --recursive tests/jobQueue",
+    "test-cache": "npm run test-cache-jobQueue && npm run test-cache-models && npm run test-cache-misc",
+    "test-cache-jobQueue": "mocha --recursive tests/cache/jobQueue",
+    "test-cache-models": "mocha --recursive tests/cache/models",
+    "test-cache-misc": "mocha tests/cache",
```

Comment:
I used the default tree reporter here instead of the dot reporter because I needed to be able to see exactly which tests were being run to fix the issue where they were being skipped. (see below)

Is there a good reason to use the dot reporter? I think it would be more useful to use the default reporter for all the tests. If not, should I change these to use the dot reporter for consistency?

Reply:
default reporter is fine ... I use it locally a lot cuz dot reporter tends to "hide" slow tests

Reply:
Can I go ahead and change it for all the other tests?

Reply:
@iamigo can you confirm?